www/src/languages.ts

```diff
@@ -32,7 +32,7 @@ export function getIsRtlFromUrl(pathname: string) {
 }
 
 export function getIsRtlFromLangCode(language: KnownLanguageCode) {
-  if (rtlLanguages.indexOf(language) !== -1) {
+  if (rtlLanguages.includes(language)) {
```

Comment:
[`@typescript-eslint/prefer-includes`](https://v8--typescript-eslint.netlify.app/rules/prefer-includes) was promoted to the [`stylistic-type-checked` config](https://v8--typescript-eslint.netlify.app/users/configs/#stylistic-type-checked) in v8.